tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -160,6 +160,12 @@ class HarnessCase(unittest.TestCase):
     started = 0
     """number: test case started timestamp"""
 
+    case_need_shield = False
+    """bool: whether needs RF-box"""
+
+    device_order = []
```

Comment:
question: does the number of devices in `device_order` must be equal to the number of golden devices in the test?

Reply:
not only golden devices, but all devices including DUT, and should match the case needed roles in topologyconfig file